Textile/Textile.sublime-syntax

```diff
@@ -1,225 +1,1111 @@
-        			":								# End name
-        			(\w[-\w_]*)						# Linkref
-        			(?=[^\w\/;]*?(<|\s|$))			# Catch closing punctuation
-        											#  and end of meta.link
+        ^
+        (p)
+        ({{textile_indentation_optional}})
+        ({{textile_alignment_optional}})
+        ({{textile_attributes_optional_class_style}})
+        (\.)
+        [ ](?=\S)
+      scope: entity.name.section.paragraph.textile
+      captures:
+        1: entity.name.tag.paragraph.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.paragraph.textile
+      push: textile-paragraph-body
+    - match: ^(?=\S)
+      push: textile-paragraph-body
+
+  textile-extended-paragraph-body:
+    - meta_scope: meta.paragraph.textile
+    - include: textile-next-block-pop
+    - include: html
+    - include: textile-inlines
+
+  textile-paragraph-body:
+    - meta_scope: meta.paragraph.textile
+    - include: textile-empty-line-pop
+    - include: html
+    - include: textile-inlines
+
+###[ BLOCKS / LISTS ]##########################################################
+
+  # https://textile-lang.com/doc/bulleted-unordered-lists
+  # https://textile-lang.com/doc/numbered-ordered-lists
+  # https://textile-lang.com/doc/definition-lists
+
+  textile-lists:
+    - match: '^(\*+)({{textile_attributes_optional}})'
+      captures:
+        1: markup.list.unnumbered.bullet.textile
+        2: markup.list.unnumbered.textile entity.name.type.textile
+      push: textile-unordered-list-item
+    - match: '^(#+)({{textile_attributes_optional}})'
+      captures:
+        1: markup.list.numbered.bullet.textile
+        2: markup.list.numbered.textile entity.name.type.textile
+      push: textile-ordered-list-item
+    - match: '^[-]'
+      scope: markup.list.definition.bullet.textile
+      push: textile-definition-list-item
+
+  textile-unordered-list-item:
+    - meta_content_scope: markup.list.unnumbered.textile
+    - include: textile-empty-line-pop
+    - match: '^(?=[-*#])'
+      pop: 1
+    - include: html
+
+  textile-ordered-list-item:
+    - meta_content_scope: markup.list.numbered.textile
+    - include: textile-empty-line-pop
+    - match: '^(?=[-*#])'
+      pop: 1
+    - include: html
+
+  textile-definition-list-item:
+    - meta_content_scope: markup.list.definition.textile
+    - include: eol-pop
+    - include: textile-definitions
+    - match: '(.*)$\n?'
+      captures:
+        1: invalid.illegal.expected-definition.textile
+      pop: 1
+
+  textile-definitions:
+    - match: '(?:[[:blank:]]*)([[:upper:]]{3,})(?:[[:blank:]]*)(:=)'
+      captures:
+        1: meta.definition.term.textile
+        2: punctuation.separator.key-value.textile
+      push: textile-definition-definition
+
+  textile-definition-definition:
+    - meta_content_scope: meta.definition.definition.textile
+    - include: merge-conflict-markers
```

Comment:
As `merge-conflict-markers` is already included by `html`, and none of the following explicit patterns seems to cause ambiguities with conflict markers, we can probably remove this include?

Or did you encounter any issues? If yes, we can probably just move `html` include further up?

```suggestion
```